integration_tests/pages/referral/on-behalf-of.ts

```diff
@@ -0,0 +1,9 @@
+import Page, { PageElement } from '../page'
+
+export default class OnBehalfOfPage extends Page {
+  constructor() {
+    super('Referrals (On Behalf Of) asdadasdwq is under construction...')
+  }
+
+  miniProfileDob = (): PageElement => cy.findByText('Date of birth').next('span')
```

Comment:
Might be best to have a miniProfile property in the Page class as it's in the main layout. Can be found with `cy.get('[data-qa=mini-profile]')`. We could then assert that the mini profile is displayed on the pages we expect rather than the DOB itself is present. That's better because the contents of the mini profile may be changed by the designers